docs/installation.rst

```diff
@@ -110,3 +110,40 @@ need to be updated. This can be achieved, for example, by installing or
 updating the ``mailcap`` package on a Red Hat distribution, ``mime-support`` on
 a Debian distribution, or by editing the keys under ``HKEY_CLASSES_ROOT`` in
 the Windows registry.
+
+Cross-Origin Request Blocked
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Debug Toolbar loads a `JavaScript module`_. Typical local development using
```

Comment:
The wording is almost the same as above but not quite. Either the JavaScript loads a module or JavaScript is loaded as modules. Both may be correct (nitpicking, sorry)